source/index.html.md

```diff
@@ -630,6 +770,35 @@ object][ApplePay-PaymentToken] for more information.
 </p>
 
 
+##### Method: `samsungpay`
+
+Samsung Pay provides the payment details as either JWE or JWE/JWS JOSE objects.
+Currently only the JWE is accepted.
+
+<dl class="dl-vertical">
+  <dt>
+    samsungpay[jwt]
+    <span class="type">
+      <a href="https://tools.ietf.org/html/rfc7516#section-3.1">
+        Compact serialized JWE token
+      </a>
+    </span>
+  </dt>
+  <dd>
+    Complete JWE token as received from Samsung Pay.
+  </dd>
+  <dt>
+    samsungpay[cek]
+    <span class="type">
+      [:base64:]{24}
+    </span>
+  </dt>
+  <dd>
+    The 128 bit symmetric AES content encryption key (CEK) used to encrypt the
+    payment token, Base64 encoded. This CEK must be extracted from the JWE
+    token.
```

Comment:
`</dd>` missing

Reply:
Nicely spotted :) I assume you ran it through the validator?

Reply:
I just stumbled upon it randomly and that reminded me to check a validator (but didn't dare to run it on the PR version; couldn't immediately figure out an easy way for the online W3 thingy) 🤷‍♂️